README.md

```diff
@@ -293,15 +299,24 @@ transfer()
     local upload_files
     local curl_output
     local awk_output
+    local current_system=$(uname -s)
+
+    # be compatible with bsd
+    if [[ "${current_system}" == *"BSD"* ]]
```

Comment:
this will fail on macos
I think we can find a single list of args that is working on both `du` variants, even at the cost of slightly changeing the exact size that will be printed (1000 vs 1024)